ArduSub/control_althold.cpp

```diff
@@ -95,21 +95,37 @@ void Sub::althold_run()
         }
     }
 
+    // Hold actual position until zero derivative is detected
+    static bool engageStopZ = true;
+    // Get last user velocity direction to check for zero derivative points
+    static float lastVelocityZ = 0;
```

Comment:
```suggestion
    static bool lastVelocityZSign = 0;
```

Reply:
We need to hold the last velocity, we can't check zero crossing without it.